app/circleSettings.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { SafeAreaView, Text, View, TouchableOpacity, Animated, ScrollView } from 'react-native';
+import Modal from "react-native-modal";
+import { styled } from 'nativewind';
+import { Button } from '../components/Buttons';
+import { Member } from '../components/Member.js';
+
+const StyledSafeArea = styled(SafeAreaView);
+const StyledView = styled(View);
+const StyledText = styled(Text);
+const StyledAnimatedView = styled(Animated.createAnimatedComponent(View));
+const StyledScrollView = styled(ScrollView);
+
+export default function Page() {
+
+	const [modalVisible, setModalVisible] = useState(false);
+
+	const [isEnabled, setIsEnabled] = useState(false);
+	const toggleSwitch = () => setIsEnabled(previousState => !previousState);
+	const togglePosition = React.useRef(new Animated.Value(1)).current;
+
+	const Trevor = "https://media.licdn.com/dms/image/C4E03AQEjKbD7qFuQJQ/profile-displayphoto-shrink_200_200/0/1574282480254?e=1701907200&v=beta&t=1BizKLULm5emiKX3xlsRq7twzFTqynOsfTlbRwqNuXI";
+
+	React.useEffect(() => {
+		Animated.timing(togglePosition, {
+		toValue: isEnabled ? 45 : 5,
+		duration: 200,
+		useNativeDriver: false,
+		}).start();
+	}, [isEnabled]);
+
+	return (
+		<StyledSafeArea className='bg-offblack border' style={{ flex: 1 }}>
+			<StyledView className='flex-1 items-center'>
+				<Button
+					btnStyles='sticky absolute left-5 top-10 bg-grey rotate-180'
+					height={'h-[60px]'}
+					width={'w-[60px]'}
+					iconSize={40}
+					icon='log-out'
+					iconColor='#F9A826'
+        			onPress={() => setModalVisible(true)}
+				/>
+				<Button
+					btnStyles='sticky absolute right-5 top-10 bg-grey'
+					height={'h-[60px]'}
+					width={'w-[60px]'}
+					iconSize={40}
+					icon='trash'
+					iconColor='#CC2500'
+				/>
+				<StyledText className='absolute top-10 text-3xl text-offwhite bg-grey text-center h-[60px] px-[35px] py-[12px] rounded-full'>
```

Comment:
This element doesn't render with rounded corners on my device. I dont know why, because it should based on the image you sent earlier. I think the best solution here is to put the StyledText inside of a StyledView that handles the background and corners